cypress/e2e/gui/validarCampoTexto.cy.js

```diff
@@ -0,0 +1,14 @@
+describe('Testes dos componentes da aplicação Engage Sphere', () => {
+  beforeEach(() => {
+    cy.intercept('GET', '**/customers**').as('getCustomers')
+    cy.visit('/customers?page=1&limit=10&size=All')
```

Comment:
Não sei como esse `cy.visit` está funcionando, visto que você está passando os parâmetros que a API espera.

Aqui deveria ser só o seguinte:

```suggestion
    cy.visit('/')
```